contrib/checkbox-provider-ce-oem/units/strict-confinement/powermanagement-strict-confinement.pxu

```diff
@@ -18,7 +14,10 @@ imports:
 requires:
   lsb.distributor_id == 'Ubuntu Core'
   snap.name == 'test-strict-confinement'
-depends: com.canonical.certification::init-boot-loop-data
+command:
+  set -e
+  rtcwake -d "${RTC_DEVICE_FILE:-rtc0}" -v -m on -s "${STRESS_BOOT_WAKEUP_DELAY:-120}" &
```

Comment:
This line can lead to just shutting down the whole machine.
The ampersand at the end doesn't guarantee that the spawned process will be scheduled on the CPU before the next line runs, rendering this a race condition.

Reply:
Hi @kissiel,
This command using argument `-m on` means it will keep the system on and read the RTC device until an alarm time appears. It won't shut down the system. However, I guess I should use `-m no` instead.

https://man7.org/linux/man-pages/man8/rtcwake.8.html